Formula/elektra.rb

```diff
@@ -0,0 +1,48 @@
+class Elektra < Formula
+  desc "Configuration Framework"
+  homepage "https://libelektra.org"
+  url "https://www.libelektra.org/ftp/elektra/releases/elektra-0.8.21.tar.gz"
+  sha256 "51892570f18d1667d0da4d0908a091e41b41c20db9835765677109a3d150cd26"
+  head "https://github.com/ElektraInitiative/libelektra.git"
+
+  option "with-qt", "Build GUI frontend"
+
+  # Build Dependencies
```

Comment:
This comment and the `# Run-Time Dependencies` one below aren't especially necessary. The tags on the deps kinda speak for themselves.